app/styles/_library/_module.footer.scss

```diff
@@ -84,21 +84,19 @@
 }
 
 .c-main-footer__nav {
-  @include media("<=xlarge") {
+  @include media("<=medium") {
```

Comment:
i think we've been trying to structure media queries as mobile first, so instead of `<=medium`, we'd want to do `>medium`, switch around to the appropriate place.